gui/default/syncthing/core/syncthingController.js

```diff
@@ -61,6 +64,10 @@ angular.module('syncthing.core')
             $scope.metricRates = (window.localStorage["metricRates"] == "true");
         } catch (exception) { }
 
+        try {
+            $scope.showRemoteGUI = (window.localStorage["showRemoteGUI"] == "true");
+        } catch (exception) { }
```

Comment:
This will reset `$scope.showRemoteGUI` to false if not present in local storage, overriding the previous default of `true`.

```suggestion
        if ("showRemoteGUI" in window.localStorage) {
            $scope.showRemoteGUI = (window.localStorage["showRemoteGUI"] == "true");
        }
```